<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsManagerWidgetLayout</class>
 <widget class="QWidget" name="GraphicsManagerWidgetLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>407</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graphics manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolBox" name="shapes_toolbox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="all_shapes_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>540</width>
        <height>270</height>
       </rect>
      </property>
      <attribute name="label">
       <string>All shapes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="rowone_hbox">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="display_label">
           <property name="text">
            <string>Display</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display_points_cbox">
           <property name="text">
            <string>Points</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display_lines_cbox">
           <property name="text">
            <string>Lines</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display_grids_cbox">
           <property name="text">
            <string>Grids</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="display_all_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Display all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hide_all_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Hide all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear_all_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Clear all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="shapes_treewidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>425</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32767</width>
           <height>32767</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>No.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Visible</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Selected</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Used for collection</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="rowthree_hbox">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="selected_label">
           <property name="text">
            <string>Selected: </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="change_color_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Change color</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="points_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>517</width>
        <height>283</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>../../../BlissFramework/Icons/Point.png</normaloff>../../../BlissFramework/Icons/Point.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Points</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="point_treewidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Index</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="points_page_layout">
         <item>
          <widget class="QLabel" name="point_create_label">
           <property name="text">
            <string>Create new point:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="create_point_start_button">
           <property name="text">
            <string>Centre</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../BlissFramework/Icons/VCRPlay2.png</normaloff>../../../BlissFramework/Icons/VCRPlay2.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="create_point_accept_button">
           <property name="text">
            <string>Accept</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../BlissFramework/Icons/ThumbUp.png</normaloff>../../../BlissFramework/Icons/ThumbUp.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="point_create_horspacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lines_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>517</width>
        <height>283</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>../../../BlissFramework/Icons/Line.png</normaloff>../../../BlissFramework/Icons/Line.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Lines</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="line_treewidget">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Index</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Start</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>End</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="create_line_button">
           <property name="text">
            <string>Create new line</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../BlissFramework/Icons/Line.png</normaloff>../../../BlissFramework/Icons/Line.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="line_hor_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="grids_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>517</width>
        <height>283</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>../../../BlissFramework/Icons/Grid.png</normaloff>../../../BlissFramework/Icons/Grid.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Grids</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="hor_spacing_label">
             <property name="text">
              <string>Horizontal spacing (μm):</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ver_spacing_label">
             <property name="text">
              <string>Vertical spacing (μm):</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="_4">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLineEdit" name="hor_spacing_ledit">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>32767</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ver_spacing_ledit">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>32767</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="_5">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="hor_move_label">
             <property name="text">
              <string>Move horizontal:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ver_move_label">
             <property name="text">
              <string>Move vertical:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="_6">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="move_left_button">
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../../BlissFramework/Icons/Left2.png</normaloff>../../../BlissFramework/Icons/Left2.png</iconset>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="move_up_button">
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../../BlissFramework/Icons/Up2.png</normaloff>../../../BlissFramework/Icons/Up2.png</iconset>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="_7">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="move_right_button">
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../../BlissFramework/Icons/Right2.png</normaloff>../../../BlissFramework/Icons/Right2.png</iconset>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="move_down_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../../BlissFramework/Icons/Down2.png</normaloff>../../../BlissFramework/Icons/Down2.png</iconset>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="spacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>91</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="grid_treewidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>425</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32767</width>
           <height>32767</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Beam width (μm)</string>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Beam height (μm)</string>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Lines</string>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Images per line</string>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="grid_create_hlayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="draw_grid_button">
           <property name="text">
            <string>Draw</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../BlissFramework/Icons/Grid.png</normaloff>../../../BlissFramework/Icons/Grid.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="grid_hor_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>91</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="remove_grid_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
